src/main/java/parsing/Parser.java

```diff
@@ -249,6 +260,28 @@ private void createCache(String filename, NodeGraph graph) {
     }
 
     /**
+     * Writes alle genomes in the string to the file given by the writer.
+     * @param gw Writer that writes the string.
+     * @param str String with the genomes.
+     */
+    private void addGenomes(BufferedWriter gw, String str) {
+        str = str.substring(str.indexOf(':') + 1);
+        str = str.substring(str.indexOf(':') + 1);
+        String[] genomeTemp = str.split(";");
+        try {
+            gw.write(genomeTemp.length + "\t");
+            for (String string : genomeTemp) {
+                gw.write(string.substring(0, string.length() - 6) + "\t");
+            }
+            gw.write("\n");
+            gw.flush();
+        } catch(IOException e) {
+            e.printStackTrace();
+            System.out.println("adding genomes failed");
+        }
+    }
+
+    /*
```

Comment:
This line was (presumably accidentally) changed resulting in a normal comment, instead of Javadoc.